internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -191,6 +200,7 @@ type CloudFormation struct {
 	regionalECRClient func(region string) imageRemover
 	region            string
 	console           progress.FileWriter
+	signalClient      signalClient
```

Comment:
nit: we can probably rename this client because we only cares about sigInt right now.
```suggestion
	sigInt      signalClient
```
The other codes will read ⬇️  and it'd be more obvious why we `cancel()` upon receiving a signal
```go
case <-cf.sigInt.NotifySignals():
cancel()
cf.sigInt.StopCatchSignals()
```